src/browser-frontend/views/browser/chrome/tabbar/tab.css

```diff
@@ -24,27 +32,37 @@ specific language governing permissions and limitations under the License.
 }
 
 .tab.selected {
-  background: var(--theme-tab-active-background);
+  background-position: var(--theme-tab-active-background-position);
+  box-shadow: var(--theme-focus-outline-style);
   color: var(--theme-tab-active-color);
 }
 
 .tab.deselected {
-  background: var(--theme-tab-inactive-background);
-  box-shadow: 1px 0 0 0 var(--theme-tab-separator-color);
-  color: var(--theme-tab-inactive-color);
+  /* react-css-modules generateAppendClassName requires that this exist */
 }
 
 .tab.initial {
   width: var(--theme-tab-initial-width);
+  height: var(--theme-tab-initial-height);
+  background-position: var(--theme-tab-inactive-background-position);
```

Comment:
I was kind of leaning towards getting rid of `.initial` and getting rid of `.deselected` in favour of making both of them the default